.github/workflows/controller-release.yml

```
@@ -0,0 +1,14 @@
+name: Controller Release
+run-name: Controller Release from ${{ github.ref_name }}${{ github.event.inputs.dry_run == 'true' && ' (dry-run)' || '' }}
```

actionlint (1.7.12)

[error] property "dry_run" is not defined in object type {}

(expression)